packages/components/src/components/input-line.css

```diff
@@ -6,7 +6,7 @@ kol-input [slot='input'] {
 	flex-grow: 1;
 }
 input:not([type='checkbox'], [type='radio']),
-select:not([multiple]) {
+select:not([multiple]):not([size]) {
```

Comment:
Geht über das Ticket hinaus, ist aber glaube ich notwendig um die Komponente mit `_rows` sinnvoll verwenden zu können?

Reply:
Ist die CSS Notation so korrekt (nie gesehen ;-)): `select:not([multiple],[size])`

Reply:
Wie ist das bei der Textarea geregelt?

Reply:
```html

    <select multiple size="2">
      <option>A</option>
      <option>A</option>
      <option>A</option>
      <option>A</option>
      <option>A</option>
</select>
```

Reply:
Beide CSS-Selektoren funktionieren, ich habe den Vorschlag übernommen da simpler 👍 

Textareas haben im CSS keine Höhe definiert, es wird also der user-agent-style angewandt, der ggf. auch die rows automatisch berücksichtigt.